hardhat.config.ts

```diff
@@ -45,7 +45,7 @@ const config: HardhatUserConfig = {
   },
   networks: {
     hardhat: {
-      allowUnlimitedContractSize: false,
```

Comment:
DnsVector has been pushed over the size limit, however I didn't want to disable the tests for it.